MinMaxTypeValues/MinMaxTypeValues/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace MinMaxTypeValues
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string byteType = "BYTE";
```

Comment:
It's not necessary to create min/max/name variables for each data type. These values are already stored in System.
Thus, we simply fill the memory with data that already exists. Please refactor it.